utils/sysfs/sysfs.go

```diff
@@ -74,6 +98,43 @@ func NewRealSysFs() SysFs {
 	return &realSysFs{}
 }
 
+func (self *realSysFs) GetNodesPaths() ([]string, error) {
```

Comment:
I would expect to see tests for these functions eventually. Taking into consideration how they are built (`realSysFs` is not used in any of them) I'm not sure if they deserve to be struct member functions. In OOP it would be just a static class with lots of static methods and in Go functions are first-class citizens.

Reply:
Now I don't have any idea how to make it better so I would like to left this as it is. Even if it is not beautiful it provides flexible mechanism to test many different cases related to sysfs structure.

Reply:
I agree - this is far from good but I don't have any better idea unfortunately.